recipes/pytim/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pytim" %}
+{% set version = "0.5.5" %}
+{% set sha256 = "6c9b4cd137d876b811430b80109f969ead85567bcf2a383be21982cddb606f44" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 1
+  skip: True  # [py3k or win]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - toolchain
+  run:
+    - python
+    - scipy
+    - numpy
+    - scikit-image
+    - cython
+    - MDAnalysis
+    - PyWavelets
+
+test:
+  imports:
+
+about:
+  home: http://github.com/Marcello-Sega/pytim
+  license: GPLv3
```

Comment:
GPL-3.0 instead of GPLv3, following the naming convention in:

https://opensource.org/licenses/alphabetical